atlasdb-config/src/main/java/com/palantir/atlasdb/factory/DefaultLockAndTimestampServiceFactory.java

```diff
@@ -341,12 +338,18 @@ private static LockAndTimestampServices getLockAndTimestampServices(
         NamespacedConjureLockWatchingService lockWatchingService = new NamespacedConjureLockWatchingService(
                 serviceProvider.getConjureLockWatchingService(), timelockNamespace);
 
-        LockWatchManagerInternal lockWatchManager =
-                LockWatchManagerImpl.create(metricsManager, schemas, lockWatchingService, cachingConfig);
-
         Supplier<InternalMultiClientConjureTimelockService> multiClientTimelockServiceSupplier =
                 getMultiClientTimelockServiceSupplier(serviceProvider);
 
+        Supplier<Optional<RequestBatchersFactory.MultiClientRequestBatchers>> requestBatcherProvider =
+                () -> timelockRequestBatcherProviders.map(batcherProviders -> ImmutableMultiClientRequestBatchers.of(
+                        batcherProviders.commitTimestamps().getBatcher(multiClientTimelockServiceSupplier),
+                        batcherProviders.startTransactions().getBatcher(multiClientTimelockServiceSupplier)));
```

Comment:
Thoughts on pushing this into `TimelockHelperServices` too? It's only used in there, and we could just pass in `timelockRequestBatcherProviders` instead.

Reply:
So I _could_, but would also need to pass in `multiClientTimelockServiceSupplier` (and thus pass in two empty optionals from `IMTS`).

I'd lean against doing this, but let me know if you'd insist on it.